dm/loader/lightning.go

```diff
@@ -283,7 +292,8 @@ func (l *LightningLoader) Process(ctx context.Context, pr chan pb.ProcessResult)
 		}
 		failpoint.Return()
 	})
-	binlog, gtid, err := getMydumpMetadata(l.cli, l.cfg, l.workerName)
+
+	binlog, gtid, err := getMydumpMetadataByExternalStorage(ctx, l.cli, l.cfg, l.cfg.LoaderConfig.Dir, l.workerName)
```

Comment:
since we have passed `l.cfg`, no need to pass `l.cfg.LoaderConfig.Dir`

Reply:
ok